Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -463,6 +463,12 @@
           "ContentsRegex": "$SEC101/047.CratesApiKey",
           "MessageArguments": { "secretKind": "Crates API key" }
         },
+        {
+          "Id": "SEC101/048",
+          "Name": "DoNotExposePlaintextSecrets/SlackWorkflow",
```

Comment:
>SlackWorkflow [](http://example.com/codeflow?start=47&length=13)

usually a rule name has Credentials, Token or something like that as a suffix, why don't we have one here?

btw - aren't we supposed to rename all 'ApiKey' rules to 'Token'? didn't we do that rename exercise elsewhere?

Reply:
Credentials = id + password, for example.
Token/ApiKey = you use that to authenticate.
For others, such as workflows/webhooks, we don't have a specific pattern.

Now, should we change all to 'Token', when we discussed about this, we were relying on the term used by the secret.

For example, we found a Nuget api key and not a nuget token.

Reply:
I think we actually converged on 'Key' elsewhere unless there was a better term.

In any case, what name do we want here? SlackWorkflowKey?